### PR TITLE
[5.0] schemaorg format json-ld

### PR DESCRIPTION
**Pull request by heelc29**
### Summary of Changes

- use same json flags for breadcrumbs like the schemaorg plugin (see also #41250)
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/modules/mod_breadcrumbs/tmpl/default.php#L120
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/plugins/system/schemaorg/src/Extension/Schemaorg.php#L457

- pretty print json like `MetasRenderer`
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/libraries/src/Document/Renderer/Html/MetasRenderer.php#L71-L72

### Testing Instructions

1. check schemaorg ouput for breadcrumbs
2. enable debug mode

### Actual result BEFORE applying this Pull Request

1.
```JSON
{"@context":"https:\/\/schema.org","@type":"BreadcrumbList","@id":"https:\/\/j50dev.localhost\/#\/schema\/BreadcrumbList\/17"
```

2.
```JSON
{"@context":"https://schema.org","@graph":[{"@type":"Person","@id":"https://j50dev.localhost/#/schema/Person/base","url":"https://j50dev.localhost/"}
```

### Expected result AFTER applying this Pull Request

1.
```JSON
{"@context":"https://schema.org","@type":"BreadcrumbList","@id":"https://j50dev.localhost/#/schema/BreadcrumbList/17"
```

2.

```JSON
{
    "@context": "https://schema.org",
    "@graph": [
        {
            "@type": "Person",
            "@id": "https://j50dev.localhost/#/schema/Person/base",
            "url": "https://j50dev.localhost/"
        }
```

### Link to documentations
Please select:
- [x] No documentation changes for docs.joomla.org needed
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
wtf. so it is never possible to have the old style breadcrumbs itemprop richdata because this is hardcoded in the tmpl even though all richdata has been removed from tmpl files everywhere else. 

:banginghead:

**Comment by heelc29** [comment]
I don't know ... I only see its linked 🤔
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/plugins/system/schemaorg/src/Extension/Schemaorg.php#L410-L415

File: plugins/system/schemaorg/src/Extension/Schemaorg.php [hunk 1]
```diff
@@ -454,7 +454,9 @@ public function onBeforeCompileHead()
         PluginHelper::importPlugin('schemaorg');
         $eventResult = $app->getDispatcher()->dispatch('onSchemaBeforeCompileHead', $event);
 
-        $schemaString = $schema->toString('JSON', ['bitmask' => JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE]);
+
+        $prettyPrint  = JDEBUG ? JSON_PRETTY_PRINT : false;
```
Comment (Fedik) [inline plugins/system/schemaorg/src/Extension/Schemaorg.php]: Please change false to 0, that seems more correct.
I know we used `false` in script options, but that was not very correct :)

Comment (heelc29) [reply plugins/system/schemaorg/src/Extension/Schemaorg.php]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f61b122ca9cb718c17eae72e98140176bed99e82<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41271">issues.joomla.org/tracker/joomla-cms/41271</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41271">issues.joomla.org/tracker/joomla-cms/41271</a>.</sub>

**Comment by HLeithner** [comment]
thanks

**Pull request merged**